guacamole-common-js/src/main/webapp/modules/SessionRecording.js

```diff
@@ -750,29 +750,38 @@ Guacamole.SessionRecording = function SessionRecording(source, refreshInterval)
             // Pull the upcoming frame
             var next = frames[currentFrame + 1];
 
-            // The position at which the next frame should be rendered, taking
-            // into account any accumulated delays from rendering frames so far
-            var nextFramePosition = next.timestamp - startVideoTimestamp + startRealTimestamp;
```

Comment:
The big problem here is that this was supposed to be a _position_ within the recording, but it's actually a browser _timestamp_, so it's going to be way larger than the delay until the next `refreshInterval` update, meaning that the timer will continuously tick up but no frames were ever rendered.

Reply:
Also, the whole idea of scheduling a frame at some different position from the actual frame position makes no sense.